test/simulator/test/org/apache/cassandra/simulator/test/EpochStressTest.java

```diff
@@ -160,7 +170,18 @@ public void manyEpochsAndAccordConverges() throws IOException
                  },
                  config -> config.nodes(3, 3)
                                  .dcs(1, 1)
-                                 .threadCount(100));
+                                 .threadCount(100),
+                 (rs, config) -> {
+                     // When the cluster defaults are used then the test will nearly 100% of the time fail to make it to 100 epochs without the epochs failing, so want to cover this in case there is a visbility violation somewhere
+                     // if the CMS retry configs are set to 1h then the failure rate becomes 10%, so more seeds will reach all 100 epochs!
+                     if (rs.decide(0.3f)) return; // 30% of the time use the cluster defaults, so have a high failure rate for TCM transactions
```

Comment:
simulator runs a single seed, so didn't think it was worth it to make `0.3` dynamic